configure.sh

```diff
@@ -141,6 +141,7 @@ MLIR_PATH=${MLIR_PATH}
 MLIR_LDFLAGS=${MLIR_LDFLAGS}
 LLVMCONFIG=${LLVM_CONFIG_BIN}
 ENABLE_COVERAGE=${ENABLE_COVERAGE}
+LD_LIBRARY_PATH=$(${LLVM_CONFIG_BIN} --libdir)
```

Comment:
Is this line really necessary? LD_LIBRARY_PATH was not defined before and is nowhere else used either now?

Reply:
The libraries are for some reason not copied/linked in the system default path when installing the MLIR packages. So, if one doesn't manually fix this on ones own machine, then the MLIR unit tests fail.

Reply:
This is very weird, you are taking the pre-compiled ones from apt repository? I would like to take a look, I think that should also be reported because it sounds like a packaging bug.

Reply:
This is also the case for MLIR-16

Reply:
I will just resolve this conversation and move on here. I have not really the appetite to figure this out.